### ISSUE TITLE
[Localization/NL_nl] 'from now' diffs are incorrect

### ISSUE THREAD

**Issue by megaxlr** [briannesbitt/Carbon#1272]
Hi, 
the "From now" diff strings in Dutch are partially incorrect. The diff for "1 day from now" should just be "tomorrow" since 1 day from now is not correct in Dutch.
Current behaviour:
```php
Carbon::setlocale('nl');
Carbon::now()->addDays(1)->diffForHumans(); // over 1 dag.
```
Expected:
```php
Carbon::setlocale('nl');
Carbon::now()->addDays(1)->diffForHumans(); // morgen
```

Looking at how the language files are structured, is it viable to fix this?

**Comment by kylekatarnls** [comment]
As the doc says: `Carbon::JUST_NOW, Carbon::ONE_DAY_WORDS and Carbon::TWO_DAY_WORDS are now only available with en and fr languages`. In fact there is also it from the last release. But each language needs the new words to be added in the lang file.

You can provide missing translation by sending a pull-request (check the IT as an example: https://github.com/briannesbitt/Carbon/pull/1263/files)

**Comment by kylekatarnls** [comment]
Will be fixed via #1273
